docs/EnterpriseScaleAnalytics-GitHubActionsDeployment.md

```diff
@@ -68,10 +68,14 @@ To begin, please open the [infra/params.dev.json](/infra/params.dev.json). In th
 | tags | Specifies the tags that you want to apply to all resources. | {`key`: `value`} |
```

Comment:
`tags`

Reply:
Let's get this done as part of another PR.